### PR TITLE
Move all PyTorch/XLA regression test on Trillium to one DAG page

### PR DESCRIPTION
**Pull request by ManfeiBai**
# Description

modify Mantaray&XLML to parse xlml tasks, merge all tests for PyTorch/XLA into one dag page

# Tests

Screenshot 2024-12-16 at 9 14 01 PM

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/mantaray/run_mantaray_jobs.py [hunk 1]
```diff
@@ -29,18 +30,41 @@
...
+  ) as dag:
+    for workload_file_name in workload_file_name_list:
       run_workload = mantaray.run_workload(
-          workload_file_name=job["file_name"],
+          workload_file_name=workload_file_name,
       )
+      run_workload
+
+  # Create a DAG for each job from maxtext
+  for job in xlml_jobs:
```
Comment (RissyRan) [inline dags/mantaray/run_mantaray_jobs.py:56]: Will this run all MaxText jobs? It will be huge. cc @gobbleturk

Comment (ManfeiBai) [reply dags/mantaray/run_mantaray_jobs.py:56]: this change want to split how to treat pytorchxla tests, and MaxText jobs should not been affected; Due to Maxtext jobs are treated in the old solution

Comment (RissyRan) [reply dags/mantaray/run_mantaray_jobs.py:56]: Thanks for the pointer of xlml file. The maxtext jobs are running before this PR.

**Review by RissyRan — APPROVED** [review]

**Review by RissyRan — COMMENTED** [review]
Please also add ownership into this file: https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/.github/CODEOWNERS. So your team could directly review them

**Comment by ManfeiBai** [comment]
> https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/.github/CODEOWNERS

Thanks, created PR for add CODEOWNERS: https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/509

**Pull request merged**